examples/ising/run.py

```diff
@@ -0,0 +1,28 @@
+import solara
+from ising.model import IsingModel
+from ising.portrayal import portray_spin
+from mesa.visualization import JupyterViz
```

Comment:
You can now import SolaraViz
```suggestion
from mesa.visualization import SolaraViz
```